spec/requests/admin/users_manage_spec.rb

```diff
@@ -110,6 +110,15 @@ def full_profile
 
       expect(user.reload.twitter_username).to eq("Twitter")
     end
+
+    it "merges an identity on a single account into the other" do
+      create(:identity, user: user, provider: "twitter")
+      create(:identity, user: user2)
+
+      post merge_admin_user_path(user.id), params: { user: { merge_user_id: user2.id } }
+
+      expect(user.identities.count).to eq 2
```

Comment:
I wonder if we should replace this to "expect that the second identity ID is contained in `user.identity_ids`, just to make sure that `update_all` merged the right identity :D